utils/docker/Dockerfile.leap.15

```diff
@@ -95,12 +105,12 @@ RUN zypper --non-interactive --gpg-auto-import-keys --no-gpg-checks ref && \
 # Monkey-patch MODULEPATH
 RUN if ! grep MODULEPATH=.*/usr/share/modules /etc/profile.d/lmod.sh; then \
         sed -e '/MODULEPATH=/s/$/:\/usr\/share\/modules/'                  \
-               /etc/profile.d/lmod.sh; \
+               /etc/profile.d/lmod.sh;                                     \
     fi
 
 RUN update-ca-certificates
 RUN zypper --non-interactive --gpg-auto-import-keys --no-gpg-checks ref && \
-    zypper --non-interactive --no-gpg-checks install --allow-unsigned-rpm \
+    zypper --non-interactive --no-gpg-checks install --allow-unsigned-rpm  \
```

Comment:
Is `--allow-unsigned-rpm` necessary here?  Doesn't the repo settings do that already?

Reply:
Not sure why the no-gpg-checks install was done that way.  It is how I found it.  My only change was whitespace.

Reply:
This install step does nothing.  Removing it.